sds/src/scanner/mod.rs

```diff
@@ -832,6 +855,95 @@ mod test {
         assert_eq!(matches.len(), 1);
     }
 
+    #[test]
+    fn test_included_keywords_match_path() {
```

Comment:
As a general guideline,  I think it's better to either have more unitary test.
Either each case in this test should be explicitly described with a comment or even better each case has is dedicated test with a very explicit name. It's way easier to debug/update/fix when they fail following a change of behaviour. It will also better scope the test and avoid having duplicated tests

Here, I would recommend splitting this test in 5 tests:
- test_included_keywords_should_match_when_find_in_path
- test_included_keywords_should_match_when_find_in_path_with_case_insensivity
- test_included_keywords_should_not_match_when_missing_in_path
...

Reply:
Yep, that's a good idea
[Split tests and move path sanitization](https://github.com/DataDog/dd-sensitive-data-scanner/pull/53/commits/4c3068390ed5fce0bcadc0f6471fa1586bda910e)